ros/sonair_evk/sonair_evk/convert_coap_payload_to_pointcloud.py

```diff
@@ -1,4 +1,6 @@
-import struct
+# Copyright (C), 2024, Sonair AS
+#
+# This software is free software, licensed according to the MIT License:https://en.wikipedia.org/wiki/MIT_License import struct
```

Comment:
Ah, nice, add this to there rest of the files as well?

Reply:
My thinking was that the other files did not really contain anything close to Sonair IP - and so really should not need such license information. I'll merge merge this now - but if you think a license term should be added to all the files I will add that in a subsequent PR.